### DISCUSSION TITLE
What license does this project use ?

### DISCUSSION THREAD

**Discussion by Zerwin** [hcpy2-0/hcpy#74] [General]
I know this project is a fork of the original https://github.com/osresearch/hcpy . However that never was published under any license as far as I can see, and I couldn't find any mention of the original owner saying anything in that direction. Therefore, can this project even have any kind of license, or was the fork even allowed ?

**Comment by pmagyar** [comment]
Hello @Zerwin,

thats a good point and after i take some research, i can say thats currently a issue. I wrote a mail directly to the original developer
(https://trmm.net/About/) and I hope I can clear it up with him.
I will post any updates here in this discussion.

Thanks for this notice!

@Meatballs1 FYI
